src/library/scala/collection/immutable/HashMap.scala

```diff
@@ -169,8 +251,18 @@ object HashMap extends ImmutableMapFactory[HashMap] with BitOperations.Int {
     }
   }
 
+  override def newBuilder[A, B]: mutable.Builder[(A, B), HashMap[A, B]] = new HashMapBuilder[A,B]
+  private class HashMapBuilder[A, B] extends mutable.MapBuilder[A, B, HashMap[A, B]](HashMap.empty) {
+    //not sure if this should be part of MapBuilder
```

Comment:
Probably, but the override wouldn't be binary compatible and there's a risk of breaking existing implementations that make `++` in `Map` call `MapBuilder`.

Reply:
Maybe we could do something in 2.15, when that branch starts